addons/website_sale_delivery/models/sale_order.py

```diff
@@ -28,7 +28,7 @@ def _compute_amount_delivery(self):
             else:
                 order.amount_delivery = sum(order.order_line.filtered('is_delivery').mapped('price_total'))
 
-    def _check_carrier_quotation(self, force_carrier_id=None):
+    def _check_carrier_quotation(self, force_carrier_id=None, ):
```

Comment:
Unnecessary diff?
In master we can pass `keep_carrier` as argument to avoid overusing the context.